src/pages/billing/Project.js

```diff
@@ -276,12 +279,17 @@ const ProjectDetail = ({ project, project: { projectName }, billingAccounts, aut
 
   useEffect(() => StateHistory.update({ projectUsers }), [projectUsers])
 
-  // Render
-  const { displayName = null } = _.find({ accountName: billingAccountName }, billingAccounts) || { displayName: 'No Access' }
+  Utils.usePollingEffect(
+    async () => _.isEmpty(getBillingProject().workspacesWithIncorrectBillingAccount) ||
```

Comment:
I'm holding the billing account in a ref as the value in the lambda never changed when i used 
```javascript
[billingProject, setBillingProject] = useState( ...
```
I have no idea why. I'd really welcome an explanation.